mercury/test_can.py

```diff
@@ -0,0 +1,10 @@
+from django.test import TestCase
+from .can import CANDecoder
+
+
+class TestCANDecoder(TestCase):
```

Comment:
I'll have to update this test to actually do something useful. Can be ignored for now.